### PR TITLE
Update jq filter for efo_otar_slim.owl

### PR DESCRIPTION
**Pull request by ehsanb29**
Update jq filter for efo_otar_slim.owl in EFO version v3.56.0.
Relevant issue: [3029](https://github.com/opentargets/issues/issues/3029)

### CODE REVIEW

**Comment by DSuveges** [comment]
@jdhayhurst , based on discussion with Zoe, EFO introduced this update sometime in April. Therefore the May release was certainly impacted, however at that time the fix was not in place.

File: modules/common/Riot.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def run_riot(
         :param owl_jq: JQ filtering for the JSON-LD conversion of the OWL file
         :return: destination file path of the conversion + filtering for the given OWL file
         """
-        path_output = Path(dir_output).joinpath(json_file)
+        path_output = os.path.join(dir_output, json_file)
         riot_outfile = random_temp_file_path(dir_output)
         riot_cmd = f"{self.riot_cmd} --output JSON-LD {owl_file} > {riot_outfile}"
         jq_cmd = f"{self.jq_cmd} -r '{owl_jq}' {riot_outfile} > {path_output}"
```
[the remark this answers is not in the document]
Comment (ehsanb29) [reply modules/common/Riot.py:62]: > I suspect _path_output_ was being serialized to _str_ here, so the command was working fine, but, when returned back to the caller on line _74_, it was being passed as a Path object, and that’s why it was not working.

Considering that the issue with `Path` is solved in the [PR 110](https://github.com/opentargets/platform-input-support/pull/110/commits/de4c8eef8c8a106b7bdb99c323e3535c4cd3055f), Can I ignore this change?

[the remark this answers is not in the document]
Comment (mbdebian) [reply modules/common/Riot.py:62]: Before moving this PR forward, you’ll need to either pull the fix through a merge from _master_ when PR 110 is merged, or cherry pick the update from that PR

**Review by mbdebian — CHANGES_REQUESTED** [review]
Thanks for the PR @ehsanb29 , would you mind getting up to speed with the master branch?

**Review by mbdebian — APPROVED** [review]
Thanks @ehsanb29

**Pull request merged**